sydent/sydent.py

```diff
@@ -184,6 +188,10 @@ def sighup(signum, stack):
             self.cfg.get('userdir', 'userdir.allowed_homeservers', '')
         ))
 
+        self.next_link_valid_regex = re.compile(
+            self.cfg.get('general', 'next_link.valid_regex')
+        )
```

Comment:
it might be good to wrap this with some exception handling so that if the regex cannot be compiled, we can give the admin a clue about where the regex is coming from.